sdk/src/metrics/aggregation/histogram_aggregation.cc

```diff
@@ -24,14 +24,14 @@ LongHistogramAggregation::LongHistogramAggregation(
   }
   else
   {
-    point_data_.boundaries_ = std::list<long>{0l, 5l, 10l, 25l, 50l, 75l, 100l, 250l, 500l, 1000l};
+    point_data_.boundaries_ =
+        std::list<double>{0.0, 5.0, 10.0, 25.0, 50.0, 75.0, 100.0, 250.0, 500.0, 1000.0};
```

Comment:
FYI https://github.com/open-telemetry/opentelemetry-specification/pull/2770 🤣

Reply:
thanks, it's good opportunity to fix the default range too. Will add the change :)

Reply:
Added.